core/sodasql/cli/cli.py

```diff
@@ -225,8 +225,12 @@ def fileify(name: str):
         first_table_scan_yml_file = None
 
         dialect = warehouse.dialect
-        tables_metadata_query = dialect.sql_tables_metadata_query()
-        rows = warehouse.sql_fetchall(tables_metadata_query)
+
+        if hasattr(dialect, "sql_tables_metadata"):
```

Comment:
We needed so that we could control what exactly rows would look like. For Spark it was not possible to have the table names in the first element of the tuple. A similar trick is done for `sql_columns_metadata_query` over `sql_columns_metadata`.